### PR TITLE
Adds confirm message 

### PR DESCRIPTION
**Pull request by pschoffer**
## Description
Adds a confirmation step before removing the previous index version when using automatic reindexing using versions.

### Plugin Updated: Search

Adds a confirmation step before removing the previous index version when using automatic reindexing using versions.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [x] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Steps to Test

1. run `dev-env exec -- wp vip-search index --using-versions --setup --indexables="post"`
2. See a confirm message before removing index

### CODE REVIEW

File: search/includes/classes/commands/class-corecommand.php [hunk 1]
```diff
@@ -45,6 +46,11 @@ private function _shift_version_after_index( $assoc_args ) {
 				WP_CLI::error( sprintf( 'Error activating next version: %s', $result->get_error_message() ) );
 			}
 
+			if ( ! $skip_confirm ) {
+				WP_CLI::confirm( '⚠️  You are about to remove previously used index version. It is advised to verify that the new version is being used before continuing. Continue?' );
```
Comment (GaryJones) [inline search/includes/classes/commands/class-corecommand.php:50]:
```suggestion
				WP_CLI::confirm( '⚠️  You are about to remove a previously used index version. It is advised to verify that the new version is being used before continuing. Continue?' );
```

File: search/includes/classes/commands/class-corecommand.php [hunk 2]
```diff
@@ -45,6 +46,11 @@ private function _shift_version_after_index( $assoc_args ) {
 				WP_CLI::error( sprintf( 'Error activating next version: %s', $result->get_error_message() ) );
 			}
 
+			if ( ! $skip_confirm ) {
+				WP_CLI::confirm( '⚠️  You are about to remove a previously used index version. It is advised to verify that the new version is being used before continuing. Continue?' );
+			}
+
+
```
Comment (pauarge) [inline search/includes/classes/commands/class-corecommand.php:53]: I'd remove one line break so we don't have two

Comment (pschoffer) [inline search/includes/classes/commands/class-corecommand.php:53]:
```suggestion
```

Comment (pschoffer) [reply search/includes/classes/commands/class-corecommand.php]: done:)

**Review by pauarge — APPROVED** [review]
LGTM

**Pull request merged**